src/Watcher.ts

```diff
@@ -108,7 +158,8 @@ export default class Watcher {
 
     // Set slider value
     if (this._style === "slider") {
-      this._dom.slider.value = value;
+      // TODO: handle non-numeric slider values
+      this._dom.slider.value = String(value);
```

Comment:
Can we file an issue for this?

Reply:
https://github.com/leopard-js/leopard/issues/168